test/Microsoft.AspNet.FileProviders.Embedded.Tests/EmbeddedFileProviderTests.cs

```diff
@@ -55,7 +55,7 @@ public void GetFileInfo_ReturnsNotFoundFileInfo_IfFileDoesNotExistUnderSpecified
         {
             // Arrange
             var provider = new EmbeddedFileProvider(GetType().GetTypeInfo().Assembly, Namespace + ".SubNamespace");
-            var expectedFileLength = new FileInfo("File.txt").Length;
```

Comment:
Yeah, looks like they are unused. Does adding `warningsAsErrors` in the project.json call these out?